src/client/sandbox/shadow-ui.js

```diff
@@ -393,8 +386,19 @@ export default class ShadowUI extends SandboxBase {
         return names.join(' ');
     }
 
-    getUIStylesheet () {
-        return nativeMethods.querySelector.call(this.document, 'link.' + SHADOW_UI_CLASS_NAME.uiStylesheet);
+    getUIStylesheetsCopyFragment () {
+        var stylesheets         = Array.prototype.slice.call(
+            nativeMethods.querySelectorAll.call(this.document, 'link.' + SHADOW_UI_CLASS_NAME.uiStylesheet));
+        var fragment = document.createDocumentFragment();
+
+        if (stylesheets.length)
+            stylesheets.forEach(stylesheet => fragment.appendChild(stylesheet.cloneNode(true)));
```

Comment:
We don't use Array.prototype methods in client code, because it can be overriden by the user code. Use `for` cycle to copy items. You will not need `Array.prototype.slice` call above as well.